susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1577,5 +1581,13 @@
          'BASECHANNEL' : 'astralinux-orel-pool-amd64', 'PKGLIST' : PKGLISTASTRALINUXOREL,
          'DEST' : DOCUMENT_ROOT + '/pub/repositories/astra/orel/bootstrap/',
          'TYPE' : 'deb'
+     },
+     'openeuler2203-x86_64-uyuni': {
```

Comment:
```suggestion
     'openeuler22.03-x86_64-uyuni': {
```
We can add a a dot here, same as we have for Ubuntu :-)